packages/x-codemod/src/v6.0.0/view-components-rename-value-prop/view-components-rename-value-prop.test.ts

```diff
@@ -0,0 +1,27 @@
+import path from 'path';
+import { expect } from 'chai';
+import jscodeshift from 'jscodeshift';
+import transform from '.';
+import readFile from '../../util/readFile';
+
+function read(fileName) {
+  return readFile(path.join(__dirname, fileName));
+}
+
+describe('v6.0.0', () => {
```

Comment:
Ideally we should/could add at least one testable item into the `preset-safe` fixtures as well, to double ensure the consistency between transformations and automatically ensure that incorrect transformation order would fail a test. 🤔

Reply:
I renamed the `value` prop of the `CalendarPicker` into `date` in the `actual.spec.js` of the `preset-safe` :+1: